diff_drive_controller/CMakeLists.txt

```diff
@@ -15,7 +15,11 @@ catkin_package(
 )
 
 include_directories(
-  include ${catkin_INCLUDE_DIRS}
+  include
+)
+
+include_directories(SYSTEM
```

Comment:
Could you elaborate on this please? What kind of warnings?

Reply:
E.g. 
```terminal
/opt/ros/kinetic/include/hardware_interface/internal/interface_manager.h:69:85: warning: type qualifiers ignored on function return type [-Wignored-qualifiers]
   static const void callConcatManagers(typename std::vector<T*>& managers, T* result)
                                                                                     ^
In file included from /opt/ros/kinetic/include/controller_interface/controller_base.h:37:0,
                 from /opt/ros/kinetic/include/controller_manager/controller_spec.h:41,
                 from /opt/ros/kinetic/include/controller_manager/controller_manager.h:36,
                 from /home/jeremiederay/workspaces/ros_controllers/src/ros_controllers/diff_drive_controller/test/diffbot.cpp:34:
/opt/ros/kinetic/include/hardware_interface/robot_hw.h:80:38: warning: unused parameter 'root_nh' [-Wunused-parameter]
   virtual bool init(ros::NodeHandle& root_nh, ros::NodeHandle &robot_hw_nh) {return true;}
                                      ^
/opt/ros/kinetic/include/hardware_interface/robot_hw.h:80:64: warning: unused parameter 'robot_hw_nh' [-Wunused-parameter]
   virtual bool init(ros::NodeHandle& root_nh, ros::NodeHandle &robot_hw_nh) {return true;}
```

Reply:
:+1:

Reply:
Yes, I've also started to do this. :+1: